### ISSUE TITLE
Missing dist

### ISSUE THREAD

**Issue by conor-mac-aoidh** [auth0/express-jwt#280]
### Description

The 7.x versions of this package contain a `package.json` which has a main pointing to `./dist`, but this folder is not included in the package.

### Reproduction

1. `npm i express-jwt@7.0.0`
2.  read the package: 
```
$ cat node_modules/express-jwt/package.json | grep main
  "main": "./dist",
```
3. list the package dir:
```
$ ls node_modules/express-jwt
bin  CHANGELOG.md  LICENSE  package.json  README.md  src  test  tsconfig.json
```

**Comment by jfromaniello** [comment]
wow that's really bad.

**Comment by jfromaniello** [comment]
Fixed in v7.4.1. Thanks for the heads up!!
